README.md

```diff
@@ -208,6 +212,22 @@ environment "production" {
 }
 ```
 
+### Consul app KV
+
+```hcl
+environment "production" {
+
+  # application name must match the file name
+  application "api-admin" {
+
+    # cleartext configuration for the application, will be written to /api-admin/threads
+    kv "threads" {
```

Comment:
we could consider a short-hand version too `kv "threads" "10"` 

if we did, it would mean `len(kvAST.Keys)` is = 2, and then value would be `kvAST.Keys[1].Token.Value().(string)`